src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="w-75 content-container">
+    <h2>Compacting</h2>
+    <mat-divider class="mt-1 mb-2"></mat-divider>
+
+    <div class="box p-4 mt-3">
+        <h4 class="subtitle">Hard compaction</h4>
+        <form [formGroup]="hardCompactionForm">
+            <div class="row">
+                <div class="col-md-3">
+                    <div class="d-flex align-items-center pt-1">
+                        <app-tooltip-icon class="mt-1" [tooltip]="MAX_SLICE_SIZE_TOOLTIP"></app-tooltip-icon>
+                        <span class="d-inline-block">Max slice size:</span>
+                    </div>
+                </div>
+                <div class="col-md-9 d-flex">
+                    <select class="form-select me-4 form-control-width" formControlName="sliceUnit">
+                        <option [value]="SliceUnit.B">B</option>
```

Comment:
Just bytes might be unnecessary from user point of view. Let's have kilobytes as a minimal suggested unit.

Reply:
Removed SliceUnit.B